src/electron/platform/android/setup/steps/prompt-configuring-port-forwarding-failed.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { AndroidSetupStepConfig } from 'electron/platform/android/setup/android-setup-steps-configs';
+
+export const promptConfiguringPortForwardingFailed: AndroidSetupStepConfig = deps => {
+    return {
+        actions: {
```

Comment:
Should have a cancel handler?

Reply:
Per offline discussion, leaving for a separate PR (other steps also missing them)